app/lib/import_export.rb

```diff
@@ -765,9 +767,10 @@ def csv_export(config, exported_data, index)
       next unless config.dig(:data, data_type, :checked).present?
 
       package = CSV.generate(headers: true) do |csv|
-        fields = config[:data][data_type][:checked]
-        csv << config[:data][data_type][:headers] || fields.map { |field| ALL_FIELDS_NAMES[data_type][field] }
-        exported_data[data_type].each do |record|
+        fields = config.dig(:data, data_type, :checked)
+        # puts config.dig(:data, data_type, :headers) || fields.map { |field| ALL_FIELDS_NAMES.dig(data_type, field) }
+        csv << (config.dig(:data, data_type, :headers) || fields.map { |field| ALL_FIELDS_NAMES.dig(data_type, field) })
```

Comment:
the lack of parenthesis here before was causing the custom export csv format to fail